mercury_app/tests.py

```diff
@@ -136,6 +214,31 @@ def test_home(self):
         self.assertEqual(response.status_code, 302)
 
 
+class SelectEventsLoggedTest(TestBase):
+
+    def setUp(self):
+        super(SelectEventsLoggedTest, self).setUp()
+
+    @patch('mercury_app.views.get_api_organization', return_value=MOCK_ORGANIZATION_API.get('organizations'))
+    @patch('mercury_app.views.get_api_events_org', return_value=MOCK_EVENT_API.get('events'))
+    def test_screen_with_one_event(self, mock_get_api_events_org, mock_get_api_organization):
```

Comment:
I'd add a more descriptive name here as `test_template_is_rendered_successfully_with_one_event_only`. Since the test just assert the status_code.